Doist/typist
# fix(factories): Allow alphanumeric IDs for suggestion nodes in `createSuggestionExtension`

## Overview

Fixes #64.

## PR Checklist




## Test plan

- Open the preview Storybook deployed to Netlify
- Open the `Rich-text → Default` story
- Add `[John Doe](mention://@user123:matrix.org)` in the `content` control (bottom)
    - [x] Observe that the editor was initialized with a mention suggestion identified
    - [x] Observe that the Markdown output shows `[John Doe](mention://@user123:matrix.org)`
